react/react1/week1/todo-tracker/src/components/ToDoList.jsx

```diff
@@ -0,0 +1,33 @@
+import TodoRows from "./ToDoRows";
+
+function ToDoList(props) {
+  const toDoListArray = [];
+  props.ds.forEach((element) => {
+    toDoListArray.push(
+      <TodoRows
+        description={element.description}
+        deadline={element.deadline}
+        key={element.id}
+      />
+    );
+  });
+
+  return (
+    <div className="toDoList">
+      <h1 className="h1">ToDo List</h1>
+      <table cellPadding="10%">
+        <thead className="tableHead">
+          <tr>
+            <th>Deacription</th>
+            <th>Deadline</th>
+          </tr>
+        </thead>
+        <tbody className="tableBody" key={toDoListArray}>
+          {toDoListArray}
```

Comment:
Instead of creating a new array you could instead use .map and do something like this:
```
      {props.ds..map((item) => {
        return (
          <TodoRows
           description={element.description}
        deadline={element.deadline}
        key={element.id}
          />
        );
      })}
```